redata/db_operations.py

```diff
@@ -22,7 +22,11 @@ def get_db_object(db_source):
         return MySQL(db_source['name'], db)
     
     raise Exception('Not supported DB')
-    
+
+def get_db_by_name(name):
+    for source_db in settings.REDATA_SOURCE_DBS:
+        if source_db['name'] == name:
+            return create_engine(source_db['db_url'])
```

Comment:
Should probably wrap this call to `create_engine()`, so it will work for Exasol too :) Eg. much like in `get_db_object()`.